playground/src/TabPane.tsx

```diff
@@ -96,4 +98,18 @@ export class TabPane extends React.Component<ITabPaneProps, ITabPaneState>  {
     this.setState({ selectedTabIndex: tabIndex });
   }
 
+  private getTabIndexFromUrl(): number {
+    const { tabs } = this.props;
+
+    let index: number = this.DEFAULT_TAB_INDEX;
+
+    tabs.forEach((tab: ITabDefinition, idx: number, _tabs: ITabDefinition[]) => {
+      const urlHashTitle: string = document.location.hash.substring(1, document.location.hash.length);
+      if (tab.title.toLowerCase() === urlHashTitle.toLowerCase()) {
+        index = idx;
```

Comment:
>idx [](start = 16, length = 3)

Our coding convention is to always spell out English words rather than using arbitrary abbreviations